lib/approvals/writers/custom_writer.rb

```diff
@@ -0,0 +1,13 @@
+module Approvals
```

Comment:
Let's not include this file--people can define the writer themselves and include it in their project. I really like this way of extending the writer!